dataView/DataView.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {hoistComponent, elemFactory} from 'hoist/core';
+import {grid} from 'hoist/grid';
+import {GridModel} from 'hoist/grid';
+import {baseCol} from 'hoist/columns/Core';
+
+@hoistComponent()
+class DataView extends Component {
+
+    constructor(props) {
+        super();
+        const {store, selection, contextMenuFn, itemFactory} = props.model;
+        this._gridModel = new GridModel({
+            store,
+            selection,
+            contextMenuFn,
+            columns: [
+                baseCol({
+                    field: 'id',
+                    flex: 1,
+                    renderElement: (props) => {
+                        return itemFactory({record: props.data});
+                    }
+                })
+            ]
+        });
+    }
+
+    render() {
+        const {rowCls, rowHeight} = this.props;
+        return grid({
+            model: this._gridModel,
+            gridOptions: {
+                headerHeight: 0,
+                rowClass: rowCls,
+                // Note: Setting a fixed row height is required for now.
+                // Ag-grid 17.1.x will introduce an 'autoHeight' property that will size rows according to content
+                rowHeight: rowHeight
```

Comment:
Hi - we recently took 17.1.0 into develop - does that get us this feature, or is it still on the planning board for them?

Reply:
Oh, I must have missed that - let me try